Euclidean_Space.R

```diff
@@ -0,0 +1,83 @@
+#Euclidean_Space
+
+library(reticulate)
+gs <- import_from_path("geomstats", path = ".")
+
+EuclideanSpace <- setRefClass("EuclideanSpace",
+                              contains = Manifold,
+                              methods = list(
+
+                                Init <- function(self, dimension){

```

Comment:
This should be `initialize` so that it is called directly when instantiating an object:

See: https://www.rdocumentation.org/packages/methods/versions/3.5.1/topics/ReferenceClasses
"Two method names are interpreted specially, initialize and finalize. If an initialize method is defined, it will be invoked when an object is generated from the class. See the discussion of method $new(...) in the section “Initialization Methods”.

Reply:
Also see "Initialization Methods" in the above link.